app/views/layouts/application.html.erb

```diff
@@ -25,6 +25,7 @@
             </div>
             <p>
               <% if logged_in? %>
+                <%= link_to 'Reset', '/articles', method: :delete, class: 'internal-link nav-link' %>
```

Comment:
Given that this `Reset` feature is relevant to the reading list itself, we could move it outside of the navigation, and next to the `Reading list` heading instead. Somewhere around here (perhaps next to the `Add by URL` link) would work great!

https://github.com/freshreader/core/blob/fe8a63848a032372d5233573ae35164711cf5918/app/views/articles/list.html.erb#L2-L9